Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs

```diff
@@ -89,5 +90,108 @@ public async Task TrackEventWithPropertiesAsync()
             }
             Assert.Equal(1, httpNetworkAdapter.CallCount);
         }
+
+        [Fact]
+        public async Task TrackEventNormalFlowAsync()
+        {
+            var httpNetworkAdapter = new HttpNetworkAdapter(expectedLogType: "event");
+            DependencyConfiguration.HttpNetworkAdapter = httpNetworkAdapter;
+
+            // Start App Center.
+            AppCenter.UnsetInstance();
+            Analytics.UnsetInstance();
+            AppCenter.LogLevel = LogLevel.Verbose;
+            AppCenter.Start(_appSecret, typeof(Analytics));
+
+            // Pause Analytics module.
+            Analytics.Pause();
```

Comment:
There are 2 problems with this test.
First it adds an unexpected feature we need to document as we are now binding publicly Pause and Resume.
Second, if you remove pause and resume, the outcome would be exactly the same as we don't wait, the batching being 3s. We would first need to wait like 4s and verify nothing has been sent. Then resume and do the rest of the test.